### ISSUE TITLE
After ten clicks, it won’t open, it will automatically close.

### ISSUE THREAD

**Issue by WMSmile** [benweet/stackedit.js#22]
After ten clicks, it won’t open, it will automatically close.
[This URL issue remains the same](https://benweet.github.io/stackedit.js/)
With Google Chrome, cache files of the same name will always increase, and more than ten will not open.

I hope you can solve it, thanks.

**Comment by WMSmile** [comment]
[image](https://postimg.cc/ZvHXbMqL)
、

**Comment by shenxiang11** [comment]
I meet this same problem.
And i had a temporarily solution.
howerver, hope @benweet can fix it officially.

I copy this source code, and remove the close function in $messageHandler;
Then add a button to close the editor.

```
this.$messageHandler = (event) => {
      console.log(event)
      if (event.origin === this.$origin && event.source === iframeEl.contentWindow) {
        switch (event.data.type) {
          case 'ready':
            // StackEdit has its own one close button
            closeButton.parentNode.removeChild(closeButton);
            break;
          case 'fileChange':
            // Trigger fileChange event
            this.$trigger('fileChange', event.data.payload);
            if (silent) {
              this.close();
            }
            break;
          case 'close':
          default:
            // remove close
        }
      }
    };
```

<img width="1276" alt="2018-11-09 6 51 09" src="https://user-images.githubusercontent.com/17292115/48258606-72d6da80-e450-11e8-855c-30dcefa66689.png">

**Comment by shenxiang11** [comment]
my chrome version  is  72.0.3602.2

**Comment by shenxiang11** [comment]
The problem seems to related to the filename, try to change the filename in event ```openFile```

@WMSmile

**Comment by WMSmile** [comment]
@shenxiang11 
If the file name is the same, the same file should be opened each time. But it created a lot of files with the same name. When the file name is the same as ten, it has this problem.

**Comment by shenxiang11** [comment]
@WMSmile 
嗯,不管名字,上限就是10个,这几天试出来了;满10个,删除了打开才没有问题

满了10个, event 会派发过来一个close的类型,然后编辑器就关闭了,像我那样直接把close方法注释虽然能打得开,但有点太hack了

**Comment by cadavre** [comment]
I have the same problem. Immediately both: iframe and silent mode stopped working.

**Comment by benweet** [comment]
Should be fixed with StackEdit v5.13.3.
Open StackEdit (https://stackedit.io) in a new tab and wait a few seconds to force upgrade.

Sorry for the trouble.
